src/SanityContext.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {
+  Dispatch,
+  SetStateAction,
+  createContext,
+  useEffect,
+  useState,
+} from 'react';
+
+import { createClient } from '@sanity/client';
+
+interface SanityData {
+  [key: string]: object[];
+}
```

Comment:
It would be nice if data from api was fully typed.

Reply:
How would I type it? We have yet to create the schemas for documents.

Reply:
It can be added later. We can't use it anyway when sanity is not ready.